examples/purchase-tester/maestro-tests/play_store_login.yaml

```diff
@@ -0,0 +1,20 @@
+appId: com.revenuecat.purchases_sample
+---
+- launchApp:
+      appId: "com.android.vending"
+      clearState: true
+- tapOn:
+      text: "Sign in"
+- tapOn:
+      # This is the id for the inputText in the play store.
+      id: "identifierId"
```

Comment:
This was obtained using `maestro hierarchy` on the relevant screen, though since it depends on the Play Store, if they change it, it's possible for this to break.